exchanges/gateio/gateio_wrapper.go

```diff
@@ -164,7 +164,7 @@ func (g *Gateio) Setup(exch *config.ExchangeConfig) error {
 		GenerateSubscriptions:            g.GenerateDefaultSubscriptions,
 		Features:                         &g.Features.Supports.WebsocketCapabilities,
 		OrderbookBufferLimit:             exch.WebsocketOrderbookBufferLimit,
-		BufferEnabled:                    true,
+		BufferEnabled:                    exch.WebsocketOrderbookBufferEnabled,
```

Comment:
SER PLS ok confirmotion for the errors ok? THANKS BYE 
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/31364354/101437651-ccc66900-3964-11eb-84a8-1a2cfabae9ea.png)

Reply:
0s at the bottom right. just check to make sure theyre actually placeholders n not some glitch

Reply:
That occurs due to the differences in item data in the output of streaming order book data via gctcli. So some blank data might be present.